components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -677,10 +683,16 @@ class PromptFeature private constructor(
      * Dismisses the select login prompt if it is active.
      */
     @VisibleForTesting
-    fun dismissLoginSelectPrompt() {
-        (activePromptRequest as? SelectLoginPrompt)?.let {
-            loginPicker?.dismissCurrentLoginSelect(it)
+    fun dismissLoginSelectPrompt(): Boolean {
+        (activePromptRequest as? SelectLoginPrompt)?.let { selectLoginPrompt ->
+            loginPicker?.let { loginPicker ->
+                if (loginPickerView?.asView()?.isVisible == true) {
+                    loginPicker.dismissCurrentLoginSelect(selectLoginPrompt)
+                    return true
+                }
+            }
         }
+        return false
```

Comment:
I think we should add tests to validate the new behaviour as now we are returning true when dismiss otherwise false, additionally we should update the docs